src/test/ui/wf/hir-wf-check-erase-regions.rs

```diff
@@ -0,0 +1,12 @@
+// Regression test for #87549.
```

Comment:
Doesn't this need some flags to use incremental or be an incremental test outright? Otherwise it's not really testing the issue, or did you manage to create a non-incremental repro?

Reply:
I thought the tests were run with `-C incremental=true` by default, if that's not the case I can add it.